src/test/java/org/cactoos/http/HtHeadTest.java

```diff
@@ -102,8 +110,54 @@ public void largeText() throws IOException {
                     )
                 )
             ).asString(),
-            Matchers.endsWith("text/plain")
+            new StringEndsWith("text/plain")
```

Comment:
@ilyakharlamov please replace this with cactoos-matchers `EndsWith` (and don't call `asString()` on the `Text`)

Reply:
@victornoel Not sure I understand. I have a `Text`, and a `EndsWith`. `EndsWith` is basically a `Matcher<String>`. How do I assert them between each other without using asString()?

Reply:
@ilyakharlamov https://github.com/llorllale/cactoos-matchers/blob/master/src/main/java/org/llorllale/cactoos/matchers/EndsWith.java is a `Matcher<Text>` :)

Reply:
@victornoel thank you, I was very confused, there are so many matchers in the classpath that do similar things but incompatible. fixed in 8915ec4